pkg/tablecodec/tablecodec.go

```diff
@@ -324,45 +322,42 @@ func DecodeRowKey(key kv.Key) (kv.Handle, error) {
 }
 
 // EncodeValue encodes a go value to bytes.
-func EncodeValue(sc *stmtctx.StatementContext, b []byte, raw types.Datum) ([]byte, error) {
+// This function may return both a valid encoded bytes and an error (actually `"pingcap/errors".ErrorGroup`). If the caller
+// expects to handle these errors according to `SQL_MODE` or other configuration, please refer to `pkg/errctx`.
+func EncodeValue(loc *time.Location, b []byte, raw types.Datum) ([]byte, error) {
 	var v types.Datum
-	err := flatten(sc, raw, &v)
+	err := flatten(loc, raw, &v)
 	if err != nil {
 		return nil, err
 	}
 
 	// `sc` is possible to be `nil` here.
```

Comment:
I think we should delete this line of comment

Reply:
Fixed.